monte-carlo/qiskit/mc_benchmark.py

```diff
@@ -505,7 +505,7 @@ def get_args():
     parser.add_argument("--skip_qubits", "-k", default=1, help="Number of qubits to skip", type=int)
     parser.add_argument("--max_circuits", "-c", default=3, help="Maximum circuit repetitions", type=int)  
     parser.add_argument("--method", "-m", default=1, help="Algorithm Method", type=int)
-    parser.add_argument("--num_state_qubits", "-nsq", default=0.0, help="Number of State Qubits", type=int)
+    parser.add_argument("--num_state_qubits", "-nsq", default=1, help="Number of State Qubits", type=int)
```

Comment:
previous default value `0.0` (floating type) is incorrect since the default type should be `int`.